geomstats/information_geometry/information_manifold.py

```diff
@@ -0,0 +1,107 @@
+"""Mixin for manifolds of probability distributions."""
```

Comment:
For consistency with geometry, shouldn't we call this file `base.py`? It may be more clearer for a user that is something to be inherited instead of instantiated.

Reply:
Good point. Done. I also renamed InformationManifold into InformationManifoldMixin to follow scikitlearn conventions